napari/_qt/qt_main_window.py

```diff
@@ -488,6 +488,14 @@ def _add_file_menu(self):
         )
         screenshot_wv.triggered.connect(self._screenshot_dialog)
 
+        clipboard = QAction(
+            trans._('Copy Screenshot to Clipboard...'), self._qt_window
+        )
+        clipboard.setStatusTip(
+            trans._('Copy screenshot of current display to the clipboard')
+        )
+        clipboard.triggered.connect(self.qt_viewer.clipboard)
```

Comment:
This is triggering the screenshot for the viewer content and not the entire window, `self.qt_viewer.clipboard` vs `self.clipboard`

Just want to make sure, so that the action text is clear `Copy Screenshot to Clipboard` sounds like it would run `self.clipboard`

Reply:
True, it does call the `qt_viewer.clipboard` method rather than the `clipboard` method. It's following the same logic as the `Screenshot` menu options. Perhaps it would make sense to change the text to `Copy Screenshot of the Canvas to Clipboard` although that is quite long menu entry.

Reply:
Indeed. If it is following the same convention as a preexisting action then we are fine :) and no change is needed.